src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+CA1839 | Performance | Info | PreferIsExactSpellingIsTrueForKnownApisAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1839)
```

Comment:
Link is no longer correct.

Reply:
Would running the codefix provider again offer a solution? Or will it generate an invalid link again?